apps/geo/filter_set.py

```diff
@@ -109,7 +114,10 @@ class Meta:
 
     def geo_area_label(self, queryset, _, value):
         if value:
-            return queryset.filter(title__icontains=value)
+            normalized_value = value.lower()
```

Comment:
Not required, `i` in `icontains` does case-insensitive search.
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#icontains

Reply:
@RobertKoikai Can you also try using this instead `unaccent__icontains`?

Reply:
@thenav56 ,yes this also worked well, plus its short & precise.